src/scripts.js

```diff
@@ -106,7 +127,7 @@ function displayWaterStatus() {
 
 function populateFriends(userFriends) {
```

Comment:
We have some functions that manipulates with user friends. This functions makes operations with datasets, so it's better to move this functions into user class. It allows us to keep scripts.js clean.

Reply:
Got it. I will add this to the User class